modules/conversion-center/module.php

```diff
@@ -296,35 +414,56 @@ private function register_links_pages_cpt() {
 			],
 		];
 
-		register_post_type( self::CPT_LINKS_PAGES, $args );
+		register_post_type( $cpt, $args );
 	}
 
 	private function has_links_pages(): bool {
 		if ( null !== $this->has_links_pages ) {
 			return $this->has_links_pages;
 		}
 
+		$this->has_links_pages = $this->has_pages(
+			self::CPT_LINKS_PAGES,
+			self::LINKS_PAGE_DOCUMENT_TYPE
+		);
+
+		return $this->has_links_pages;
+	}
+
+	private function has_contact_pages(): bool {
+		if ( null !== $this->has_contact_pages ) {
+			return $this->has_contact_pages;
+		}
+
+		$this->has_contact_pages = $this->has_pages(
+			self::CPT_CONTACT_PAGES,
+			self::CONTACT_PAGE_DOCUMENT_TYPE
+		);
+
+		return $this->has_contact_pages;
+	}
```

Comment:
We should probably split these into separate classes... 
`$this->register_contact_pages_cpt/$this->register_links_pages_cpt
`
--> 

`$this->links->register_cpt / $this->cb->register_cpt`

But we can do it if there's time left.

Reply:
@louiswol94 I think this should be architectured in a way that can be re used throughout the code base. Happy to give it a shot if we have time. Probably should live more in the Document rather than in the module, but not sure how/when modules are initialized.